packages/vats/src/core/startWalletFactory.js

```diff
@@ -0,0 +1,135 @@
+// @ts-check
+import { E, Far } from '@endo/far';
+import { deeplyFulfilled } from '@endo/marshal';
+
+import * as BRIDGE_ID from '../bridge-ids.js';
+import { makeStorageNodeChild } from '../lib-chainStorage.js';
+import { makeMyAddressNameAdmin, PowerFlags } from './basic-behaviors.js';
+
+const { details: X } = assert;
+
+/**
+ * @param {ERef<ZoeService>} zoe
+ * @param {Installation<import('@agoric/smart-wallet/src/walletFactory').start>} inst
+ * @typedef {Awaited<ReturnType<typeof startFactoryInstance>>} WalletFactoryStartResult
+ */
+// eslint-disable-next-line no-unused-vars
+const startFactoryInstance = (zoe, inst) => E(zoe).startInstance(inst);
```

Comment:
```suggestion
/**
 * @typedef {Awaited<ReturnType<import('@agoric/smart-wallet/src/walletFactory').start>>} WalletFactoryStartResult
 */
```

Reply:
Though why define that type? The start result only has a creator facet so you could put that in the space instead of https://github.com/Agoric/agoric-sdk/blob/eb5b1f5b0635680081d708e1928c8ee089b61c80/packages/vats/src/core/types.js#L218-L220

Reply:
The result from `E(zoe).startInstance()` also has an admin facet, which is useful for upgrade. I suppose the motivation for `startFactoryInstance` was under-documented.